src/components/App.js

```diff
@@ -9,10 +9,9 @@
 // bonus: have player decide how many cards to play with 
```

Comment:
Actually You don't need to move `App` under components folder since it's kind of acting as a `root` component, but all the other components should go under `components` folder for sure.